020-LinuxFundamentals/Student/Challenge-04.md

```diff
@@ -0,0 +1,38 @@
+# Challenge 04 - File contents
+
+[< Previous Challenge](./Challenge-03.md) - **[Home](../README.md)** - [Next Challenge >](./Challenge-05.md)
+
+## Description
+
+In this challenge let's learn about file content manipulation and discover how to count file lines, display specific lines from a file, and more.
+
+- Display the first 10 lines of `/etc/resolv.conf`
+- Display the last 5 lines of `/etc/crontab`
+- Create a file named `count.log` with this content:
+
+    One<br>
+    Two<br>
+    Three<br>
+    Four<br>
+    Five
+
+- Use `cp` to make a backup of this file to `count.bkp`
+- Use `cat` to make a backup of this file saving as `cat-count.log`
```

Comment:
add a comma:
Use `cat` to make a backup of this file, saving as `cat-count.log`